File: POO/TP2/Practica Python/EJ3/Alumno.py
class Alumno:
    __nombre = str
    __apellido = str
    __dni = int
    
    def __init__(self):
        pass
    
    @classmethod
    def iniciar(cls):
        alumno = cls.__new__(cls)
        return alumno
    
    @classmethod
    def iniciar_con_nom_ape(cls,nombre,apellido):
        alumno = cls.__new__(cls)
        alumno.__nombre = nombre
        alumno.__apellido = apellido
        return alumno
    
    
    def setNombre(self, val):
        self.__nombre = val
    
    
    def setApellido(self, val):
        self.__apellido = val
    
    
    def setDni(self, val):
        self.__dni = val
    
    
    def getNombreYapellido(self):
        return f"{self.__nombre} {self.__apellido}"
    
    def getDni(self):
        return self.__dni

File: POO/TP2/Practica Java/EJ3/Alumno.java
package EJ3;

public class Alumno {
    private String nombre = "";
    private String apellido = "";
    private Integer dni = 0;

    public String getNombre() {
        return nombre;
    }

    public void setNombre(String nombre) {
        this.nombre = nombre;
    }

    public String getApellido() {
        return apellido;
    }

    public void setApellido(String apellido) {
        this.apellido = apellido;
    }

    public Integer getDni() {
        return dni;
    }

    public void setDni(Integer dni) {
        this.dni = dni;
    }

    public Alumno() {

    }

    public Alumno(String nombre, String apellido) {
        this.nombre = nombre;
        this.apellido = apellido;
    }

    public String getNombreYapellido() {
        return this.nombre + " " + this.apellido;
    }
}
